langlib/lang.xml/src/main/ballerina/xml.bal

```diff
@@ -371,6 +536,11 @@ public isolated function filter(xml<ItemType> x, @isolatedParam function(ItemTyp
 # This parses the string using the `content` production of the
 # XML 1.0 Recommendation.
 #
+# ```ballerina
+# string bookStr = "<book>Ballerina</book><book>Java</book>";
+# xml bookXml = check xml:fromString(bookStr);
```

Comment:
```suggestion
# check xml:fromString("<book>Ballerina</book><book>Java</book>") ⇒ 
```

Reply:
Updated in https://github.com/ballerina-platform/ballerina-lang/pull/38981/commits/7b4dfa608b04064393e46211524556d9850dfddf